qiskit/qpy/binary_io/value.py

```diff
@@ -50,20 +56,132 @@ def _write_parameter_vec(file_obj, obj):
     file_obj.write(name_bytes)
 
 
-def _write_parameter_expression(file_obj, obj, use_symengine, *, version):
-    if use_symengine:
-        expr_bytes = obj._symbol_expr.__reduce__()[1][0]
+def _encode_replay_entry(inst, file_obj, version, side=False):
```

Comment:
If `side` is used to mark the right-side operand, I think it would be clearer to say it explicitly, maybe:

```suggestion
def _encode_replay_entry(inst, file_obj, version, r_side=False):
```
(again, minor suggestion, it just took me a bit to go and check which side it was). If I confused the side, the same would apply to the left.

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/13356/commits/11f65adbc08690ffe3cb3587814ec1274905b111